pandas/tests/frame/apply/test_frame_apply.py

```diff
@@ -1550,3 +1576,88 @@ def test_apply_empty_list_reduce():
     result = df.apply(lambda x: [], result_type="reduce")
     expected = pd.Series({"a": [], "b": []}, dtype=object)
     tm.assert_series_equal(result, expected)
+
+
```

Comment:
maybe let's create test_frame_transform.py for these (and other transform tests)